src/T2DMap.cpp

```diff
@@ -3942,8 +3996,8 @@ void T2DMap::mouseMoveEvent(QMouseEvent* event)
             return;
         }
 
-        dx = mx - pR->x;
-        dy = my - pR->y;
+        int dx = qRound((event->pos().x() / mTX) + mOx - (xspan / 2.0) + 1.0) - pR->x;
```

Comment:
What's this change do?

Reply:
I thought I was fixing-up some integer maths on float/double numbers - the code in this class is ~riddled~ _example-rich_ :wink: with it - also does some algebraic rearrangement to get rid of some unneeded intermediates {e.g. the local floats `tx` and `ty` are exactly replaceable with the member floats `mTX` and `mTY`, though a search and replace on "ty" has some unfortunate clashes with some keywords and other classes/methods...}.  Also `qRound(...)` rounds `float`/`double` argument to nearest `int`.